### ISSUE TITLE
Wireframes

### ISSUE THREAD

**Issue by jpasx** [linuxmint/mint-design#5]
Hi everyone.

Can we begin to discuss the wireframes for home?
I think a simple web is what you are looking for.
Responsive design. This wireframe is for desktop. When we know what we want, we will make wireframes for other resolutions.

Logo - top left.
Menu - top right.
A full-screen image with a "Donwload" call to action.
Sponsors.
Boxes with different flavors.
Google ads.
Footer

What we need to add? What we need to remove?
![linuxmint wireframe jaime 1 0](https://cloud.githubusercontent.com/assets/2758331/7029573/dd3e5d78-dd5e-11e4-93de-31730a7ce534.JPG)

**Comment by promovis** [comment]
Congratulations webdev team for this project (I'd like to join it) and jpasx for this first mock-up. Pretty straightforward and close to the existing website, might not be a "shocking" change for the visitors. Responsive, of course. For the desktop version (min-width 1200px is a lot of screen) I kindly suggest to keep a sidebar, in my (subjective) opinion the left sidebar looks somehow better than the right one, what do you think?

**Comment by emorc** [comment]
I like and I agree @promovis the side bar must be and maybe add a small follow bar into.
I think we need to add a call to action to donate and just below the sponsors section an announcement area.

By the way I'm mexican and my english is not the best  :(.

**Comment by Sirikon** [comment]
The wireframe looks great! And I think that the sidebars in landing pages are ugly, think this as the landing page of a product, what was the last landing page with sidebar that you saw?

A landing page needs to "sell" the product (in this case the user must fall in love with the project), and to achieve that we need all the available width, with graphics that reflect the project's spirit and passion.

Think the landing page as a story:

(First view, the big image with the Download button) "Hey... LinuxMint... Looks great"

(Good stuff and description of the project, short and easy to understand descriptions) "its really good, and interesting, tell me more"

(Trusted by companies a.k.a. sponsors) "Those are important companies... I don't know some of them but hey, are companies, this is a serious project! I want it!"

(All this depends on you, donate to the linuxmint project!) "I should donate something, this is a great work and deserves a donation"

(Again download button) "Let's download it!"

(Footer)

**Comment by Sirikon** [comment]
Another suggestion: In the last three years, the worst month of donations, LinuxMint got 4.000$, are Google Ads really necesary? How many money gets the project with them? I think that a google ad in a landing page of a project with a big donate button is something not correct.

**Comment by klavs** [comment]
Nice work, @jpasx! As you asked for possible additions and removals, I will write down my notes. I encourage you and everybody else to have a constructive argument with me about it.

Let's start with Google adds. I definitely thing they should not be present in home page (as @Sirikon already stated). Also, I have to say that I am definitely not the person to decide about this.

Ok, let's get back to the header of the page. I've been thinking about this for last few days.
First of all, I think there is a need for a common header menu across all linuxmint sites. Let's take http://blog.linuxmint.com/ as an example. There is no easy way to get back from blogs to the main site. The same goes for http://community.linuxmint.com/. In my opinion there needs to be a common header menu to navigate across all linuxmint sites.
Then there is an argument about how many menu items should be there. I think there should be two-level menu. The main linuxmint.com site would only show the first level of menus. By clicking on them, you could get to another page, which would then contain also the second level menus. Let's say there is first-level menu item called "Community". By clicking on that, you would be directed to community.linuxmint.com, where you could choose one of second-level menus (for example, the ones that are now present in community portal). This would allow users to navigate back from sub-sites back to the main site.
The hardest part for me is deciding, what will be those first-level items. I think there should be no more than four of them. If we can, let's fit all items under only three sections.

In my opinion, the landing page should be designed for casual user, even those who do not yet know what Linux is. That makes me think that the main menu should be sectioned by kinds of users. For example, "casual user", "curious user" and "experienced user".
Casual user should be pointed to the download of linuxmint. I would even say that he should get only one choice for distros - Cinnamon edition (as it is covered by the same people and can be supported and updated internally). We should make his thought process as straightforward as possible - come to site, download an image, install Cinnamon edition, run it. (From this point on, user should also have a clear path to get help from Linux Mint community).
Curious user is the user which has taken his time and installed Linux Mint or at least has a serious interest in it. For this kind of user, we should provide non-technical forums and blogs, and community portal. Also, this kind of user is advanced enough to try out other Linux Mint editions (at his own risk).
Experienced user is a user who is willing to put his effort in making Linux Mint better. Developers, testers, designers, translators, etc. go in here.

**Comment by Sirikon** [comment]
@klavs "the main menu should be sectioned by kinds of users. For example, "casual user", "curious user" and "experienced user"" I think that's an interesting approach, but maybe not the most correct.

I as user, when see a project, the first thing I check out is the project's repo activity, At start I'll not contribute to the project without knowing if it's interesting, but I'll check the repo to see the activity and see if the project is alive. This is a behaviour that would be out of that approach, that's a nice menu organization, but not with that names.

Maybe the best way is do something similar to "Casual User", "Curious User" and "Experienced User"

 * Download (Goes to a download page, with the default desktop configuration as the most big and important button for the casual user, and other download links for the curious user)**
 * Community (Forums and blogs for the curious user)
 * Contribute (Development guidelines, how to test, design guidelines, translation websites, etc, for the experienced user)

This way we organize all the elements in a similar way, but with names that invites others users to get in, not block them ("I'm casual user but I want to see the test area, why is this only for experienced users? Im not dumb!")

** Anyway, there will be a big "Download" button with a direct download link of the ISO with the default desktop configuration.

**Comment by klavs** [comment]
@Sirikon, you got the point! These were not supposed to be the names of sections. I do not have an oppinion about the names yet. Your suggestion sounds good.

I can see the reasons for your behavior, but I belive that Linux Mint should also be accessible for users that do not even know how software is made.

**Comment by Sirikon** [comment]
@klavs Of course, thats why there're two download buttons and I think in LinuxMint not as a Debian-Based Linux distribution with X license etcetcetc, I think in LinuxMint as a product to sell, so it should look great to any type of user.

Any other suggestion to make this more friendly for non-technical users guys? :smiley: 

(Also @jpasx , I'm working in a wireframe with some suggestions :wink:  )

**Comment by JaimePascual** [comment]
Hi everyone.

@promovis @emorc I think that sidebar is not a good idea for home but if you upload a wireframe with your ideas maybe @Sirikon and me will change our opinion ;)

@promovis I have based the wireframe in major parts that I've seen in the current home (in my point of view). It's just a starting point.

@Sirikon @klavs  Maybe google ads is not necesary but actually is in home and I guess that is necessary until the contrary is indicated

@Sirikon @klavs About names of sections in my wirefreme I put a few names like test. I think the final composition of menu should be discussed in later stages.

@Sirikon I want see your wireframe.

@all Would it be possible to end this week with designing wireframes to move to the next stage next week? What do you think?

**Comment by Sirikon** [comment]
@JaimePascual (and everyone) Here you have the wireframe: https://moqups.com/Sirikon/sZkPTJyE (I'm trying to export it to PNG so everyone can see it more easy)

About the Google Ads, maybe @clefebvre could say something about this.

**Comment by promovis** [comment]
Dear friends,
The passion & brainstorming involved here is amazing already, i can almost see the future LinuxMint state-of-the-art website. I must confess I am a Linux Mint addict, after a forced landing from Windows and some bouncing between a few distros like Ubuntu & Peppermint, and never regret it but loving it more and more because of its speed, stability, usability etc.
@Sirikon: no sidebars for homepage, if they're futile here might be useful elsewhere. But why not mix the Google ads with donate button or sponsors? I suppose the homepage is receiving heavy traffic (maybe the heaviest) so the ads on the homepage might be quite productive (i'd REALLY like to see the stats from Google Analytics and Adsense... at least for the homepage :). We can make places for these ads and decide (or let a qualified person decide) if there will be ads or not in the end.
@JaimePascual your wireframe is a very good starting point and started this great debate, well done! As @klavs & @Sirikon put it, the header should be common among sections, user friendly and based (organized) on functionality rather than on user levels (which can be subjective, misleading or hurting some feelings...) As for the download, I tried both 32 and 64 bit (on the appropriate machines) Rebecca Mate, Cinnamon and KDE, which one should be the "default download"? Hard to say, should the user decide according to his particular conditions.

**Comment by Sirikon** [comment]
@promovis I think that people are sick of ads, if we put an ad near to the "Donate" button, this button will be affected negatively, because the Ads have something like a negativity aura. I think that there's more money lost by not-donating because of an ad than the earnings of that ad. (Some data from Google Analytics and AdSense would be really usefull, yes)

Of course in other parts of the webpage sidebars will be so usefull :),

**Comment by JaimePascual** [comment]
I really like your design @Sirikon . I do not like "community photo" section. I already know that Mint is very important to the community but I think a picture of this kind does nothing at home.
I made another wireframe based on your design. I think we should put sponsors in home.
And What do you think about a box for people to subscribe and receive the latest news?
@promovis I agree you in which one should be the "default download"? I think that  this should be user election. Then Download call to action should be point to donwload page?
Although not yet said, I have ruled out a search button because I think people who come to the home does not "know" what they want to search.
![linuxmint wireframe 1 1](https://cloud.githubusercontent.com/assets/2758331/7064499/67446a34-deaf-11e4-898d-0ee5beeb9de1.JPG)

**Comment by Sirikon** [comment]
@JaimePascual When I said "Trusted companies" also meant sponsors, my fault :yum: and the "latest news" area to subscribe is a great idea.

About the "Default Download", I think that it should be the Cinnamon version, and thanks to the userAgent in Javascript, detect the computer's architecture and give the 32bit download to 32bit users and 64bit download to 64bit users. (We could even detect the navigator and, if the user is using Konqueror for example, offer the KDE version, but this is just a idea/possibility)

**Comment by klavs** [comment]
Minor note on menu: I think the download button does not fit there as it is already visible at the same viewport. Now that elementary.io has changed their menu, we can afford to put News section instead of Download.

**Comment by dalcde** [comment]
I don't think we are legally allowed to say that we are "trusted by" the sponsors, since they have not explicitly said that they trust us. It's just that the decide to give us money. Maybe they simply think Mint is the lesser of many evils :)

**Comment by Sirikon** [comment]
@klavs The idea is that the big download button in the center of the first area will be a direct link to the default ISO, and the "Download" in the menu will show the downloads page, with more versions available to download. Also the "Download" in the menu will be always visible as the topbar could be in a fixed position (always on top of the screen)

**Comment by Sirikon** [comment]
@dalcde Would be nice to know in which companies is LinuxMint being used, also ask to those sponsors if we could say that, IMO the words "Trusted by the companies" is really strong when "selling" a project like this.

**Comment by klavs** [comment]
@Sirikon, ok, the idea is clear to me, but imagine telling a random person: "Go to linuxmint.com and press 'Download'. No not that 'Download', the other 'Download'!"

**Comment by Sirikon** [comment]
@klavs Hm... That's right, then just remove the Download button on the topbar or replace it with something?

**Comment by klavs** [comment]
I think, we could discuss the functionality of "Download" call-to-action button. Should it download immediately or link to download page, where Cinnamon edition is the primary option and other options are not that emphasized?

**Comment by Sirikon** [comment]
@klavs In that case, better link to the download page

**Comment by promovis** [comment]
I think the "Download" section in the main menu is better, since it's available site-wide, so the homepage call-to-action button can be reserved to Donations, instead of Downloads. @klavs mentioned elementary.io, I think they have an interesting approach, combining both functions under the Download button, although I'd say the visitor should be able to donate without downloading and vice versa, IMHO

**Comment by JaimePascual** [comment]
@promovis I believe that raising money is not the main function of the home. From my point of view the main function must be convert visitors into downloads .

**Comment by klavs** [comment]
I agree to @JaimePascual's opinion.
There is a need for call to action element, and it needs to be download, not donate. Though, there could be a smaller text under it, indicating that we need donations.

**Comment by Sirikon** [comment]
@promovis @klavs Making the call-to-action button a Donate button, IMO, is a huge mistake, this way we're trying to get money before the user even know that LinuxMint is, and this is a negative experience, the donation shouldn't even appear near the call-to-action button, could be a possibility to put it in the topbar as the last menu entry for example.

**Comment by promovis** [comment]
You're right, Mint has its own identity & surpass Ubuntu both in quality as well as popularity, being amongst the most popular desktop OSes in the world and the recommended choices for Linux newcomers. The new website must show and sustain these aspects and enlarge the user base before attracting sponsors or donations. What about a testimonial section with both individual and corporate user testimonials? And maybe a FAQ section as well.

**Comment by Sirikon** [comment]
@promovis We could include a "Testimonials" section in the home page, with a link to more testimonials

**Comment by JaimePascual** [comment]
Considering the suggestions, I think we could move to the next stage from this wireframe.

![linuxmint wireframe 1 2](https://cloud.githubusercontent.com/assets/2758331/7102079/76d2582c-e075-11e4-868d-0023e0eb24f2.JPG)

Next stage would be convert the wireframe to image. Add colors, names to sections and menu, etc... I don´t like make images. I like write HTML and CSS  directly but maybe it's better create the images first.
What do you think?
Can we discuss first colors palette?

**Comment by JaimePascual** [comment]
This is my first picture. Only the top of the design. We're going discussing menus, colors, fonts, etc.
Honestly I do not really like the background image. Does anyone have a good and nice capture of your desktop or any other suggestions?
Basing the design on a green primary color (# 84cc3a) this is the palette that suggests colourco.de http://colourco.de/triad/5/%2387cf3e
I think we've added too much content to the web. When everything is finished we will see ....

![linux mint image v1 0](https://cloud.githubusercontent.com/assets/2758331/7169447/e178f926-e3c5-11e4-9634-472cdb895b1a.jpg)

If someone wants the xcf file here you can download https://www.dropbox.com/s/mjkhy5gs7kg7nxo/linux%20mint%20image%20v1.0.xcf?dl=0

**Comment by JaimePascual** [comment]
Version 1.1

![linuxmint image v1 1](https://cloud.githubusercontent.com/assets/2758331/7286681/ff242a12-e94d-11e4-9cab-05b0899580eb.jpg)

**Comment by dalcde** [comment]
I quite like the general layout, but the image selection can be better :) The top bar also looks a bit plain imo. Arch linux does quite a good job in that respect: https://www.archlinux.org/

**Comment by JaimePascual** [comment]
@dalcde Everything can be improved. They are very early versions. This is going to be putting ideas and choosing the best. Each of your suggestions are welcome. :)

**Comment by JaimePascual** [comment]
Version 1.2 (background image and arrangement of the elements  changed)

We need more powerful images. Perhaps this arrangement of the elements is smarter?

![linux mint v1 2](https://cloud.githubusercontent.com/assets/2758331/7435614/4e98defa-f045-11e4-9475-1ea1f7d40e78.jpg)

**Comment by JulienItard** [comment]
Hi guys, what do you think of this kind of template : https://creativemarket.com/Leegrant/178940-Evoke-web-app-landing-page-PSD

**Comment by SerafDosSantos** [comment]
This was going well until *boom!*... I like most of the wireframes. Would need some more work to be done. But not by me, I'm no designer.

I think we should more focus on financing for the new website. Like if we can get a few more bucks for the product, why not? Donations and some CD selling should be more in the front. Like Ubuntu for instance. They're proposing a contribution before downloading the product. I'm sure they are getting a few more bucks because of the suggested fee. Wouldn't hurt Mint to have more money.

**Comment by SerafDosSantos** [comment]
@JaimePascual your wireframes are good but the design needs a lot more work.

**Comment by SerafDosSantos** [comment]
Some idea about a common template:
![linuxmint com-pencil](https://cloud.githubusercontent.com/assets/3285430/13002999/e66cd7d6-d140-11e5-8ccd-297b4af0c368.png)

**Comment by SerafDosSantos** [comment]
As I'm going through these wireframes I see that they are generally offering a main download access but instead of adding a further step with a page in between, leading to some confusion with the 4 Mint variants to choose from, why not do as it is done right now and offer the 4 variants as download options right away.

![page_homepage 0](https://cloud.githubusercontent.com/assets/3285430/13028659/1713d9cc-d243-11e5-9b1f-9e0631cb4044.png)

**Comment by SerafDosSantos** [comment]
Here's some more ideas, this time for the blog, as blog.linuxmint.com and segfault are pretty simple to mockup and straight forward to revamp.

**Home**
![blog linuxmint com_-_home](https://cloud.githubusercontent.com/assets/3285430/13031983/f2bba236-d2af-11e5-9f24-055056769bb5.png)

**A Post**
![blog linuxmint com_-_post](https://cloud.githubusercontent.com/assets/3285430/13031985/fa0b1f6c-d2af-11e5-8d4e-147a25271479.png)

**Comment by SerafDosSantos** [comment]
My wireframes use a 950px with for content with a fluid with for top (green) header and footer. They are inspired by actual designs and content components.

**Comment by SerafDosSantos** [comment]
After analysing the pages of linuxmint.com I came to the conclusion that in order to minimize the revamp of the whole site, if we redesigned mainly the CSS and minimally the pages, the whole actual concept of the site could be kept. Largelly changing the CSS, some images (mostly icons) and some formating of the HTML and the whole site would be changed without major rehaul.

Here's a wireframe example of the common pages of linuxmint.com:

![page_linuxmint com_-_common_page 4 3](https://cloud.githubusercontent.com/assets/3285430/13038071/818529c8-d35b-11e5-85a6-61acfec81f1c.png)

**Comment by SerafDosSantos** [comment]
Of couse another option could be to use Wordpress as a main Website and why not integrate blog.linuxmint.com in it as a whole site managed by wp CMS. A viable option.

**Comment by SerafDosSantos** [comment]
Here's some basic wireframe idea for the community.linuxmint.com website:

![page_community linuxmint com_-_common_page 5](https://cloud.githubusercontent.com/assets/3285430/13242056/f4ef9f1a-d9be-11e5-9b5d-da7623bcb83e.png)

**Comment by nyrrad** [comment]
I feel as though the 960px width grid system is somewhat outdated and we should focus on providing a responsive solution that isn't so tight on large format screens. While i do agree that it would make a revamp fast i don't thing the ending result would be where we want to go.  Multi device design is it should go with ~50% of the web being on something that is not a desktop.

**Comment by nyrrad** [comment]
My biggest concern with the redesign is how to we present the ads in a way that doesn't break the fidelity of the design itself. Ads can be obtuse and off putting. If we can come up with a consistent solution concerning those i think it'll make the entire process easier.

**Comment by clefebvre** [comment]
Hi everyone,

This is dying a bit. It's mostly my fault for not being here more often and it's also because efforts are spread out, here on the forums and directly via email.

We started using slack for development and it's helped us a lot. I create a new design team there and sent you an invite. Hopefully this can boost things a bit and make it easier for us to work together.

Other than the website, I'd love it if we managed to create a core design team within Linux Mint. We need design and artwork so often when developing things..

I could find the email addresses for some of you. If you didn't get an invite and you're interested in being part of this, please let me know at root@linuxmint.com.

**Comment by StivenTC** [comment]
@clefebvre 
Hi!
Am I still in time to join the group?
I hope it's not too late.
